src/pages/components/localemodal.mdx

```diff
@@ -21,8 +21,26 @@ import ResourceLinks from "components/ResourceLinks";
 
 ## Overview
 
-The Locale modal provides the option to select a geographic regions and languages. This modal is triggered by the Locale Selector button in the Footer. The user's current location is indicated at the top of the modal.
+The Locale modal provides the option to select a geographic regions and languages. This modal is triggered by the Locale Selector button in the Footer. The user's current location is indicated at the top left of the modal.
 
-![Default](../../images/component/locale-modal/default-locale-modal.jpg)
+<Row>
+<Column colMd={8} colLg={8}>
+
+![Default](../../images/component/locale-modal/locale-modal-default.png)
+
+</Column>
+</Row>
+<Caption>The first state of the modal displays region options.</Caption>
+
+The user can click or tap a region to view a searchable list of locations and languages. They can select an option to reload the page as the corresponding location and language.
+
+<Row>
+<Column colMd={8} colLg={8}>
+
+![Default](../../images/component/locale-modal/locale-modal-country.png)
+
+</Column>
+</Row>
+<Caption>The second state of the modal displays a searchable list of location and region pairings.</Caption>
```

Comment:
@RichKummer, should this be `location and language pairings`?

Reply:
It should be, region is the first state. Just pushed an update with the new language.